tests/zfs-tests/tests/functional/cli_user/misc/dbufstat_001_pos.ksh

```diff
@@ -37,4 +37,6 @@ while [[ $i -lt ${#args[*]} ]]; do
         ((i = i + 1))
 done
 
+log_must eval "dbufstat.py -F object=10,dbc=1,pool=testpool > /dev/null"
```

Comment:
Can you add a quick comment describing what this does?  Also, should it be `pool=$TESTPOOL`?